haystack/core/pipeline/base.py

```diff
@@ -669,7 +669,13 @@ def outputs(self, include_components_with_connected_outputs: bool = False) -> Di
         }
         return outputs
 
-    def show(self, server_url: str = "https://mermaid.ink", params: Optional[dict] = None, timeout: int = 30) -> None:
+    def show(
+        self,
+        server_url: str = "https://mermaid.ink",
+        super_component_expansion: bool = False,
```

Comment:
We should add `super_component_expansion` at the bottom of this function otherwise inserting it in the middle is a breaking change.